gsmphys/radiation_gases.f

```diff
@@ -1022,9 +1022,9 @@ subroutine getgases                                               &
         tmp = raddeg / resco2
         do i = 1, IMAX
           xlon1 = xlon(i)
-          if ( xlon1 < 0.0 ) xlon1 = xlon1 + con_pi  ! if xlon in -pi->pi, convert to 0->2pi
-          xlat1 = hfpi - xlat(i)                     ! if xlat in pi/2 -> -pi/2 range
-!note     xlat1 = xlat(i)                            ! if xlat in 0 -> pi range
+          if ( xlon1 < 0.0 ) xlon1 = xlon1 + 2*con_pi  ! if xlon in -pi->pi, convert to 0->2pi
```

Comment:
Reading the comment/note, the original code is correct.  In order to take something in the range of `-pi->pi` and convert to the range `0->2pi`, you only need to add `pi`.

Reply:
I think if we're keeping the origin constant and only changing the negative values we need to add 2 pi. Dunno why the compilation is failing, though; it works on Gaea

Reply:
After our discussion, I agree with what is being done here.  They aren't rotating the whole sphere, but keeping the origin at zero.  For the lats, they are recasting pi/2 (90N) -> -pi/2 (90S) as 0 (0N)-> pi (180S).  So I'll approve this one.